### PR TITLE
Change default Discord Channel for help

### PR DESCRIPTION
**Pull request by ChrisJefferson**
There is no #help channel on discord. I am guessing that #contribute is the best channel for people wanting to contribute, although I am guessing.

### CODE REVIEW

**Comment by ChrisJefferson** [comment]
Although maybe this wants to point to Zulip?

File: src/liballoc/raw_vec.rs [hunk 1]
```diff
@@ -150,6 +150,7 @@ impl<T, A: AllocRef> RawVec<T, A> {
             alloc_guard(layout.size()).unwrap_or_else(|_| capacity_overflow());
 
             let memory = alloc.alloc(layout, init).unwrap_or_else(|_| handle_alloc_error(layout));
+//            println!("{} {} {}\n", layout, capacity, memory);
```
Comment (nikomatsakis) [inline src/liballoc/raw_vec.rs:153]: this diff looks unrelated

File: README.md [hunk 2]
```diff
@@ -247,7 +247,7 @@ To contribute to Rust, please see [CONTRIBUTING](CONTRIBUTING.md).
 Most real-time collaboration happens in a variety of channels on the
 [Rust Discord server][rust-discord], with channels dedicated for getting help,
 community, documentation, and all major contribution areas in the Rust ecosystem.
-A good place to ask for help would be the #help channel.
+A good place to ask for help would be the #contribute channel.
```
Comment (nikomatsakis) [inline README.md:250]: Yeah, I guess that most accurately we would want to point people to either Discord or Zulip, depending on what sort of contribution they have in mind, though some groups (e.g., embedded) are elsewhere.

Comment (nikomatsakis) [reply README.md:250]: At least adding a link to Zulip seems good, though I suppose the ideal would be to list out the various teams at each spot, and I'm a bit reluctant to add that info here for fear of it getting outdated. But we should have some central resource, *probably* organized through the teams repo?

**Comment by Elinvynia** [comment]
Ping from triage:
@ChrisJefferson Hello! Could you please have a look at the review requests? Let us know if you are having issues or if you don't have the time!
